sdk/src/main/java/com/oursky/authgear/latte/LatteLink.kt

```diff
@@ -0,0 +1,48 @@
+package com.oursky.authgear.latte
+
+import android.app.Application
+import android.content.Intent
+import android.net.Uri
+import com.oursky.authgear.getOrigin
+import com.oursky.authgear.getQueryList
+
+object LatteLink {
+    const val BROADCAST_ACTION_LINK_RECEIVED = "com.oursky.authgear.latte.linkReceived"
+    interface LinkHandler {
+        suspend fun handle(latte: Latte)
+    }
+
+    private class ResetLinkHandler(
+        private val query: List<Pair<String, String>>
+    ) : LinkHandler {
+
+        override suspend fun handle(latte: Latte) {
+            val handle = latte.resetPassword(query)
+            handle.finish()
+        }
+    }
+
+    fun getAppLinkHandler(
+        intent: Intent,
+        appLinkOrigin: Uri,
+        rewriteAppLinkOrigin: Uri?
```

Comment:
What's this parameter used for?

Reply:
It will be used by login link, see
https://github.com/authgear/authgear-sdk-ios/blob/2023C01/Sources/Latte%2BUniversalLink.swift#L59